builtin/providers/azurerm/resourceid.go

```diff
@@ -104,3 +104,12 @@ func parseNetworkSecurityGroupName(networkSecurityGroupId string) (string, error
 
 	return id.Path["networkSecurityGroups"], nil
 }
+
+func parseRouteTableName(routeTableId string) (string, error) {
+	id, err := parseAzureResourceID(routeTableId)
+	if err != nil {
+		return "", fmt.Errorf("[ERROR] Unable to parse Route Table ID '%s': %+v", routeTableId, err)
```

Comment:
If the error contains more than just the error string, %+v will print fields as well. There is a [similar example](https://blog.golang.org/error-handling-and-go).